src/content/docs/mlops/bring-your-own/getting-started-byo.mdx

```diff
@@ -0,0 +1,101 @@
+metaDescription: "Start getting value from your machine learning model data in just a few minutes within New Relic"
+---
+
+This is a guide to getting started with New Relic's bring your own data. You'll learn how to install, run, and experiment with _bring your own data_ , or BYOD, and start monitoring the performance of your machine learning models.
+
+
+## Quick start [#quick-start]
+Using BYOD makes it easy to monitor your machine learning models in 3 main steps:
+
+```python
+# 1: Initialize the monitoring
+ml_monitor = MLPerformanceMonitoring(...)
+
+# 2: Add your algorithm
+y = my_model.predict(X)
+
+# 3: Record your data
+ml_monitor.record_inference_data(X, y)
+```
+
+Use this example in [colab](https://colab.research.google.com/github/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb) and easily try an end-to-end example of model monitoring.
+
+
+## Installation [#installation]
+
+Installation is straightforward and similar to any python library
+
+```python
+pip install git+https://github.com/newrelic-experimental/ml-performance-monitoring.git
+```
+
+
+## Implementation [#implementation]
+
+This guide will take you step by step for everything needed to start monitoring your Machine Learning models
+
+### 1. Set Your Environment Variable
+
+[Get your License key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) (also referenced as `ingest - license`) and set it as environment variable: `NEW_RELIC_INSERT_KEY`. [Click here](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key) for more details and instructions.
+Are you reporting data to the New Relic EU region? click [here](https://github.com/newrelic-experimental/ml-performance-monitoring#eu-account-users) for more instructions.
+
+### 2. Import package
+
+```python
+from ml_performance_monitoring.monitor import MLPerformanceMonitoring
+```
+
+###  3. Create model monitor
+
+```python
+metadata = {"environment": "notebook"}
+model_version = "1.0"
+features_columns, labels_columns = (
+    ["feature_1", "feature_2", "feature_3", "feature_4"],
+    ["target"],
+)
+
+ml_monitor = MLPerformanceMonitoring(
+    insert_key=None,  # set the environment variable NEW_RELIC_INSERT_KEY or send your insert key here
+    model_name="My stunning model",
+    metadata=metadata,
+    features_columns=features_columns,
+    labels_columns=labels_columns,
+    label_type="numeric",
+    model_version=model_version
+)
+```
+
+### 4. Do your thing
+
+```python
+y = my_model.predict(X)
+```
+
+### 5. Record
+```python
+ml_performence_monitor_model.record_inference_data(X, y)
+```
+
+### 6. Monitor and alert
+Done! Check your application in the New Relic UI to see the real time data.
+ 
+
+## Examples [#examples]
+We created these notebooks in Google colab so you can try them out:
+  1. Try out an XGBoost model on Boston housing prices dataset. Open in colab.
```

Comment:
Missing links (look at the [docs](https://docs.google.com/document/d/1QjSfxOr1Xv1fs3t3tyV8fob9230d_i7cqQT1CDzr9YI/edit#)

Reply:
done